site_root/input/ImplementationGuide-NCPI-Project-Forge.json

```diff
@@ -141,6 +141,62 @@
         "reference": {
           "reference": "StructureDefinition/age-at-event"
         }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet phenotype-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/phenotype-codes",
+        "reference": {
+          "reference": "ValueSet/phenotype-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge CodeSystem hp",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge CodeSystem/hp",
+        "reference": {
+          "reference": "CodeSystem/hp"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet hpo-observation-type-codes",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/hpo-observation-type-codes",
+        "reference": {
+          "reference": "ValueSet/hpo-observation-type-codes"
+        }
+      },
+      {
+        "description": "NCPI Project Forge CodeSystem snomed-ct",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge CodeSystem/snomed-ct",
+        "reference": {
+          "reference": "CodeSystem/snomed-ct"
+        }
+      },
+      {
+        "description": "NCPI Project Forge StructureDefinition ncpi-phenotype, Base: Observation",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge StructureDefinition/ncpi-phenotype",
+        "reference": {
+          "reference": "StructureDefinition/ncpi-phenotype"
+        }
+      },
+      {
+        "description": "NCPI Project Forge ValueSet ncpi-observation-interpretation",
+        "exampleBoolean": false,
+        "name": "NCPI Project Forge ValueSet/ncpi-observation-interpretation",
+        "reference": {
+          "reference": "ValueSet/ncpi-observation-interpretation"
+        }
+      },
+      {
+        "description": "NCPI Project Forge Observation ph-1, Profiles: ncpi-phenotype",
+        "exampleBoolean": true,
+        "name": "NCPI Project Forge Observation/ph-1",
+        "reference": {
+          "reference": "Observation/ph-1"
+        }
       }
```

Comment:
Sorry to be too picky, but can we change `"ph-1"` to `"ph-001"` to make it consistent with other example resources' ID convention?